### PR TITLE
Showing final build size

### PR DESCRIPTION
**Pull request by junwon1022**
This pull request regards part of the issue #12 .

Now the terminal shows the final build size after executing "teachbooks build /path/to/book".
The changes were made in the 'build' method in main.py file. 

Currently the function only applies when building the book, but a different command such as 'teachbooks inspect' (as @rlanzafame has mentioned) could be implemented later by extending from the current code.

image

### CODE REVIEW

File: teachbooks/cli/main.py [hunk 1]
```diff
@@ -24,11 +21,12 @@ def build(ctx, path_source, publish, release, process_only):
     """Pre-process book contents and run Jupyter Book build command"""
     from teachbooks.release import make_publish
     from jupyter_book.cli.main import build as jupyter_book_build
+    import os
```
Comment (Tom-van-Woudenberg) [inline teachbooks/cli/main.py:24]: Shouldn't `os` be a dependency in the toml file if we use it here?

**Review by Tom-van-Woudenberg — COMMENTED** [review]
One comment, didn't test it yet

File: teachbooks/cli/main.py (hunk 1 above)
Comment (junwon1022) [reply teachbooks/cli/main.py:24]: That's an unused import that isn't used anymore. I used it previously to test some stuff, but now they're not needed anymore and therefore deleted. Thank you for pointing it out!

**Review by Tom-van-Woudenberg — APPROVED** [review]
Works, tested here: https://github.com/TeachBooks/Sandbox/actions/runs/12183083970/job/33983654152#step:10:93

**Pull request merged**